apps/collabora_docker.sh

```diff
@@ -142,12 +142,16 @@ then
     occ_command app:remove onlyoffice
 fi
 
-# shellcheck disable=2034,2059
-true
+# Collabora Docker URL (collabora.sh
+SUBDOMAIN=$(input_box "Collabora subdomain eg: office.yourdomain.com\n\nNOTE: This domain must be different than your Nextcloud domain. They can however be hosted on the same server, but would require seperate DNS entries.")
+# Nextcloud Main Domain (collabora.sh)
+NCDOMAIN=$(occ_command_no_check config:system:get overwrite.cli.url | sed 's#https://##;s#/##')
```

Comment:
Please use | instead of # as we do that everywhere else.

Reply:
fixed